Sources/Workflow/Models/Workflow.swift

```diff
@@ -286,7 +286,7 @@ extension Workflow where F.WorkflowOutput == Never {
     public func thenProceed<FR: FlowRepresentable>(with type: FR.Type,
                                                    launchStyle: LaunchStyle = .default,
                                                    flowPersistence: @escaping @autoclosure () -> FlowPersistence = .default) -> Workflow<FR> where FR.WorkflowInput == Never {
-        let wf = Workflow<FR>(first)
+        let wf = Workflow<FR>(first?.copy())
```

Comment:
This `.copy()` seems to be the crux of the memory leak and the most important part of these `thenProceed`s.  Would it make sense to extract it as a preferred method for creating a fluent describer(?) of the workflow?

Spitballing, but something like:
```suggestion
        let wf = Workflow<FR>(fluently: first)
```

Reply:
Hrm, I think the `copy()` thing wasn't clear to you. The problem is that if you pass a direct reference to `first` with the new `deinit` it just gets cleaned up, so like all the proceeds don't work. 

I am unclear what `fluently` means here? Especially since it's not fluent? Another potential fix would be in the `LinkedList` intializer to make it auto-copy, rather than telling each calling instance of it to copy.

Reply:
I think you'll find my solution satisfactory